policyengine_canada/parameters/gov/provinces/bc/benefits/bcfb/temporary_enhancement/phase_out_rate_high_income.yaml

```diff
@@ -0,0 +1,21 @@
+description: British Columbia phases family benefit temporary enhancement amount out at this rate, based on high family net income.
+brackets:
+  - threshold:
+      2021-07-01: 0
```

Comment:
@PavelMakarchuk Hi Pavel,  for this part, I know we usually start at the first day of January. However, according to the definition of [family benefit temporary enhancement](https://www2.gov.bc.ca/gov/content/family-social-supports/affordability/family-benefit), we need to compare the previous year family net income with the threshold, which the period is from July to June. I would like to ask if it's okay the period here is 2021-07-01?

Reply:
we will have to make the period month and include a max month parameter 

This will cap the MONTHS_IN_YEAR function which will be multiplied by the parameter